docgenhtml/plugins/org.polarsys.capella.docgen/src/org/polarsys/capella/docgen/util/StringUtil.java

```diff
@@ -56,571 +56,498 @@
-
-		// To generate header one of the first or second title must be set
-		boolean generateTableHeader = (firstColTitle != null && !firstColTitle.isEmpty())
-				|| (secondColTitle != null && !secondColTitle.isEmpty());
-
-		firstColTitle = firstColTitle == null ? "" : firstColTitle;
-		secondColTitle = secondColTitle == null ? "" : secondColTitle;
-
-		StringBuilder stringBuilder = new StringBuilder();
-		stringBuilder.append("<table>"); //$NON-NLS-1$
-		if (generateTableHeader) {
-			stringBuilder.append("<tr>").append("<th>"); //$NON-NLS-1$ //$NON-NLS-2$
-			stringBuilder.append(firstColTitle).append("</th>"); // End first col //$NON-NLS-1$
-
-			stringBuilder.append("<th>"); //$NON-NLS-1$
-			stringBuilder.append(secondColTitle);
-			stringBuilder.append("</th></tr>"); // End second col and the first row //$NON-NLS-1$
-		}
-
-		// Generate the table content
-		for (Entry<String, String> e : map.entrySet()) {
-			stringBuilder.append("<tr>"); //$NON-NLS-1$
-			stringBuilder.append("<td>").append(e.getKey()).append("</td><td>").append(e.getValue()).append("</td>"); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
-			stringBuilder.append("</tr>"); //$NON-NLS-1$
-		}
-		stringBuilder.append("</table>"); //$NON-NLS-1$
-
-		return stringBuilder.toString();
-	}
-
-	/**
-	 * @param stringSet
-	 *            must be not null
-	 * @return true if the stringSet contains a not empty or null value string,
-	 *         otherwise false
-	 */
-	public static boolean containsNotEmptyString(Collection<String> stringSet) {
-
-		for (String string : stringSet) {
-			if (string != null && !string.isEmpty()) {
-				return true;
-			}
-		}
-		return false;
-	}
+  public static final String ELEMENT_LINK_REGEX = "hlink://(.+)";
+  public static final String REGEX = "<[\\s]*[aA][\\s]+href[\\s]*=[\\s]*\"([^>]+)\"[\\s]*>(.*?)<[\\s]*/[\\s]*[aA][\\s]*>";
+  public static final String REGEX_FILEPATH = "<[aA][\\s]+href=\"(.+?)\">";
+  public static final String ERROR_CPY = "Error during project relative file copy: {0}";
+  public static final String ERROR_COPY_PART_2 = " referenced in the description of the element: {0} "; //$NON-NLS-1$
+  public static final String ERROR_DECODE_FILE_LOCATION = "Error while decoding file location: {0}";
+  public static final String ERROR_READ_FILE_LOCATION = "Error while reading decoded file location: {0}";
+  public static final String WARNING_IMAGE_SERIALIZATION_FAILED = "Image serialization failed, default to keep img serialized value for element if id: {0}";
+  public static final String WARNING_NETWORK_IMAGE_NO_TREATMENT = "Network images are left as is and not copied to the generated documentation: {0}. Element id: {1}";
+  public static final String ERROR_STRING_PATH_URI_UNENCODE = "String to URI unencode failed with string: {0}";
+  public static final String IMAGES_FOLDER = "/images/";
+  public static final String ERROR_HYPERLINK_OBJECT_UNREACHABLE = "Hyperlink to object {0} in {1} cannot be reached";
```

Comment:
if these constant are used only in DefaultImageHandler, it is better to move them there